package.json

```
@@ -3,6 +3,9 @@
   "name": "jsonpath-plus",
   "version": "8.0.0",
   "type": "module",
+  "bin": {
+    "jsonp-cli": "./dist/jsonp-cli.js"
```

```suggestion
    "jsonp-cli": "./dist/jsonp-cli.js",
    "jsonpath": "./dist/jsonp-cli.js"
```

I think it's easier to remember with at least one more directly evoking the package name